Assets/MixedRealityToolkit/_Core/Definitions/InputSystem/MixedRealityInputActionRulesProfile.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Core.Definitions.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Core.Definitions.InputSystem
+{
+    [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Mixed Reality Input Action Rules Profile", fileName = "MixedRealityInputActionRulesProfile", order = (int)CreateProfileMenuItemIndices.InputActionRules)]
+    public class MixedRealityInputActionRulesProfile : ScriptableObject
+    {
+        [SerializeField]
+        private InputActionRuleDigital[] inputActionRulesDigital;
```

Comment:
Will need to check these in the inspector.  But would be better if this was hidden, like with the interaction definitions.  So the user simply selects an action and the rule is stored based on the axis of that action maybe?

Reply:
I'm not sure I understand.